docs/storefront/catalyst/getting-started/guides/customer-group-component.mdx

```diff
@@ -0,0 +1,105 @@
+
+Here you can see we have two groups: **Group A** and **Group B**.
+
+Now you need to assign some customers to your desired groups so you can see what they will see in your store. If you don’t have customers to work with, you can follow this [Customer Storefront Accounts](https://support.bigcommerce.com/s/article/Customer-Account-Creation?language=en_US) support article.
+
+![Customers dashboard](https://storage.googleapis.com/bigcommerce-production-dev-center/images/catalyst/customer-group-component/customers-dashboard.png)
+
+We have three customers here named Peter J, Moon D and Suzy T. In the **Group** column, we’ve assigned Suzy to **Group B**, Moon to **Group A**, and Peter to **No Group**.
+
+<Callout type="info"> 
+Make sure you click **Save** in each row when making those modifications.
+</Callout>
+
+## Working with the Customer Group Slot in Makeswift
+
+Once done, go back to your channel page and click on **Edit in Makeswift** to begin using Makeswift for your store.
+
+![Catalyst Edit in Makeswift button](https://storage.googleapis.com/bigcommerce-production-dev-center/images/catalyst/customer-group-component/catalyst-edit-in-makeswift-button.png)
+
+Switch to your [development environment](https://docs.makeswift.com/developer/guides/environments). Now in the Home page, let’s add a new section. Open up the component drawer and drop in the **Section** component into the page.
+
+<Video src={'https://storage.googleapis.com/bigcommerce-production-dev-center/videos/catalyst/customer-group-component/section-component.mp4'} />
+
+The goal here is to add a dynamic section on the page with content that changes based on the Customer Group. From here on, we’ll refer to those dynamic pieces of content as “group slots”. So, let’s drag in the **Customer Group Slot** component.
+
+<Video src={'https://storage.googleapis.com/bigcommerce-production-dev-center/videos/catalyst/customer-group-component/customer-group-slot.mp4'} />
+
+The controls for this component on the right handle the following:
+
+- **Selected groups**: Choose from a list of available customer groups and more that you’d like to create custom content for. By default, we provide the following additional selections:
+- **Simulated group:** This will control which available groups you would like to individually simulate in the editor. The additional selections are included in this control including:
+
+    By default, the **No group** selection is included, which includes target customers that are not logged in or have “No group” selected as their BigCommerce group.
+
+Once that’s finished, let’s add “Group A” to the control for **Selected groups**, and simulate it with the **Simulate group** control. Once simulated, let's add the text “A big discount for group A” that only Group A customers can see.  
+
+
+<Video src={'https://storage.googleapis.com/bigcommerce-production-dev-center/videos/catalyst/customer-group-component/group-A.mp4'}/>
```

Comment:
Can we add a note callout that say "You will need to start searching for groups before they will populate in the dropdown."